src/components/shapes/helpers.js

```diff
@@ -53,7 +53,7 @@ exports.extractPathCoords = function(path, paramsToUse, isRaw) {
     return extractedCoordinates;
 };
 
-exports.getDataToPixel = function(gd, axis, isVertical, refType) {
+exports.getDataToPixel = function(gd, axis, shift, isVertical, refType) {
```

Comment:
We could drop the 4th argument (isVertical) and get it from the `axis._id.chartAt(0) === 'y'`.

Reply:
This is unfortunately not possible, because it could also be vertical if axis is undefined (when the xref or yref is paper).
That is probably why this param was there before already.

Reply:
Thanks for the note.
Resolving...